src/features/chart/components/Chart.tsx

```diff
@@ -53,17 +54,22 @@ const formatDate = (isoDate, ChartRange) => {
 export const Chart = () => {
   const dispatch = useDispatch();
   const { selectedChartData, selectedChartRange } = useSelector(
-    state => state.chartData
+    (state: any) => state.chartData
   );
-  const { selectedStockTicker } = useSelector(state => state.stockTickerData);
-  const updateChartRange = stock => dispatch(updateChartAction(stock));
-  const onClickHandler = event => {
-    updateChartRange(event.target.value);
+  const { selectedStockTicker } = useSelector(
+    (state: any) => state.stockTickerData
+  );
+  const updateChartRange = (stock: any) =>
+    dispatch(updateChartRangeAction(stock));
+  const onClickHandler: React.MouseEventHandler<HTMLButtonElement> = event => {
+    updateChartRange(event.currentTarget.value);
   };
 
+  console.log(selectedChartData);
+
   const chartData =
     selectedChartData &&
-    selectedChartData.map(data => ({
+    selectedChartData.map((data: any) => ({
```

Comment:
You shouldn't need the `: any` here, either.

You can _partially_ type your `useSelector` whilst you lack a state shape:
```
const { selectedChartData, selectedChartRange } = useSelector<any, ChartState>(state => state.chartData);
```
You can then later change this to:
```
const { selectedChartData, selectedChartRange } = useSelector((state: AppState) => state.chartData);
```
when you have a `AppState` type for your whole redux state.